### DISCUSSION TITLE
Faucet

### DISCUSSION THREAD

**Discussion by BobboFuxx** [ping-pub/explorer#596] [General]
I’d like to use the faucet feature in testnet.ping.pub for Prysm, chainID prysm-devnet-1. The faucet.md is a bit vague on the address used for the faucet and how to add it.

**Comment by liangping** [comment]
enable faucet in config.json
```
    "faucet": {
        "amount": "5000000uatom,50000000uside",
        "ip_limit": "1",
        "address_limit": "10",
        "fees": "1000uatom"
    }
```

and funding the faucet address on faucet page.

**Reply by BobboFuxx** [reply]
Do i add the here do i create the faucet account? do i just add “account”:”prysm1”…. then find said address?

**Comment by liangping** [comment]
The address is created on your faucet page. You just need to fund it.
